website/docs/r/dev_test_global_shutdown_schedule.html.markdown

```diff
@@ -0,0 +1,140 @@
+---
+subcategory: "Dev Test"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_dev_test_global_shutdown_schedule"
+description: |-
+    Manages automated shutdown schedules for Azure Resource Manager VMs outside of Dev Test Labs.
+---
+
+# azurerm_dev_test_schedule
+
+Manages automated shutdown schedules for Azure Resource Manager VMs outside of Dev Test Labs.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "sample" {
+  name     = "sample-rg"
+  location = "eastus"
+}
+
+resource "azurerm_virtual_network" "sample" {
+  name                = "sample-vnet"
+  address_space       = ["10.0.0.0/16"]
+  location            = "${azurerm_resource_group.sample.location}"
+  resource_group_name = "${azurerm_resource_group.sample.name}"
+}
+
+resource "azurerm_subnet" "sample" {
+  name                 = "sample-subnet"
+  resource_group_name  = "${azurerm_resource_group.sample.name}"
+  virtual_network_name = "${azurerm_virtual_network.sample.name}"
+  address_prefix       = "10.0.2.0/24"
+}
+
+resource "azurerm_network_interface" "sample" {
+  name                = "sample-nic"
+  location            = "${azurerm_resource_group.sample.location}"
+  resource_group_name = "${azurerm_resource_group.sample.name}"
+
+  ip_configuration {
+    name                          = "testconfiguration1"
+    subnet_id                     = "${azurerm_subnet.sample.id}"
+    private_ip_address_allocation = "Dynamic"
+  }
+}
+
+resource "azurerm_virtual_machine" "sample" {
```

Comment:
Could we change this to the new linux/windows VM resource?

Reply:
Done.